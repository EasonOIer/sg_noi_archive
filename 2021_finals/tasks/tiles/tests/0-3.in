14 2
..............
..............
..............
2 2 11
2 1 14
