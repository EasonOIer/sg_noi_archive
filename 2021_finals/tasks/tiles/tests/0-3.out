47177097
254767228
